crates/agglayer-node/src/node.rs

```diff
@@ -86,7 +86,9 @@ impl Node {
             }
         };
 
-        let aggregator_task: AggregatorNotifier<_> =
+        let certifier_aggregator_task: AggregatorNotifier<_> =
+            config.certificate_orchestrator.prover.clone().try_into()?;
+        let epoch_packing_aggregator_task: AggregatorNotifier<_> =
```

Comment:
You can create the notifier once and put it into an `Arc`
```suggestion
        let notifier: AggregatorNotifier<_> = Arc::new(config.certificate_orchestrator.prover.clone().try_into()?);
```

Reply:
hm but they should be different I believe since the epoch packer has to be for `AggregatorNotifier<Proof>` (eventually for the aggregation proof) while certifier has to be `AggregatorNotifier<Certificate>` (for the leaf proof)

That said the epoch packer implementing the AggregatorNotifier is probably useless for now given that no proof aggregation will be done in the near future. So probably just keeping it like this in this PR and refactoring the epoch packer in the PR which will add the settlement tx to the L1?